src/site/site.xml

```diff
@@ -84,7 +84,6 @@
                 <item name="Mailing Lists" href="/mail.html" />
                 <item name="Contributing" href="/contribute.html" />
                 <item name="Guidelines" href="/guidelines.html" />
-                <item name="Wiki" href="http://wiki.apache.org/james" />
```

Comment:
shoudn't we delete this wiki, too?

Reply:
I will create the INFRA ticket once it is deployed & merged.